Makefile

```diff
@@ -346,16 +338,12 @@ e2e-local:
 ci: dep-vendor-only check-fmt lint generate check-local-changes unit integration e2e-compile docker-build
 
 # Run e2e tests in a dedicated cluster.
-ci-e2e: run-deployer
-	$(MAKE) IMG_SUFFIX=-ci install-crds apply-psp e2e
+ci-e2e: dep-vendor-only run-deployer install-crds apply-psp e2e
 
 run-deployer: dep-vendor-only build-deployer
 	./hack/deployer/deployer execute --plans-file hack/deployer/config/plans.yml --run-config-file run-config.yml
 
-ci-release: export GO_TAGS = release
-ci-release: export LICENSE_PUBKEY = $(ROOT_DIR)/build/ci/license.key
```

Comment:
Why you removed that?

Reply:
GO_TAGS is defined now [here](https://github.com/elastic/cloud-on-k8s/pull/1610/files#diff-09bab207dbf0a581c36e2dfdf631ca66R36), as to the LICENSE_PUBKEY I removed it because:
```
$ grep -Ri LICENSE_PUBKEY *
$ grep -Ri license.key *
$
```

But now I see I missed the usage in license-initializer. I'll fix, good catch.